devicon.json

```diff
@@ -4248,6 +4248,29 @@
         "color": "#1FA6CA",
         "aliases": []
     },
+    {
+        "name": "materializecss",
+        "altnames": [
+            "materialize",
+            "materialize-css",
+            "materialize css"
+        ],
+        "tags": [
+            "css framework",
+            "css",
+            "design",
+            "material-design",
+            "framework"
+        ],
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": []
```

Comment:
Please add a `plain` version of the icon to use it as a font.
Check our [**Wiki**](https://github.com/devicons/devicon/wiki/Example-of-Submitting-Icons-to-the-Repository) for more information about that. 🙂

You should also check the available [**SVG versions**](https://github.com/devicons/devicon/wiki/SVG-Versions).


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient